examples/hcp-vault/py/requirements.txt

```diff
@@ -0,0 +1 @@
+pulumi>=3.0.0,<4.0.0
```

Comment:
it might also want pulumi-hcp no?

Reply:
Not until a package has been officially published, I think. The testing framework end up downloading these requirements, then overlaying the code from the current commit for testing. I tried adding the dependency here, but it just fails because PyPI doesn't have the package yet.

I can follow up on this after the first version has been published.